packages/react/src/components/CardLink/__stories__/CardLink.stories.js

```diff
@@ -4,46 +4,58 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
+import { boolean, text } from '@storybook/addon-knobs';
 import ArrowRight20 from '@carbon/icons-react/es/arrow--right/20';
 import CardLink from '../CardLink';
+import Error20 from '@carbon/icons-react/es/error/20';
 import React from 'react';
 import readme from '../README.stories.mdx';
-import { text } from '@storybook/addon-knobs';
+
+const getBaseKnobs = ({ groupId }) => {
+  const disabled = boolean('disabled', false);
+  const iconStyle = disabled ? Error20 : ArrowRight20;
+  return {
+    card: {
+      copy: text(
+        'Card Heading:',
+        'Explore AI use cases in all industries',
+        groupId
+      ),
+      cta: {
+        href: text('Card href:', 'https://www.example.com', groupId),
+        icon: {
+          src: iconStyle,
+        },
+      },
+    },
+    disabled: disabled,
```

Comment:
This is showing up in a separate tab, can you take a look at the configuration?

<img width="487" alt="Screen Shot 2020-06-02 at 5 57 19 PM" src="https://user-images.githubusercontent.com/1641214/83574244-15efb900-a4fb-11ea-89dd-b5c1ea73ab18.png">